pkg/images/loader.go

```diff
@@ -14,15 +14,15 @@ func FromFile(filePath string) (Images, error) {
 	{
 		yamlFile, err = ioutil.ReadFile(filePath)
 		if err != nil {
-			return nil, microerror.Maskf(err, "could not read file %s: #%v ", filePath)
+			return nil, microerror.Mask(err)
```

Comment:
Maybe we should mask `executionFailedError` with message like:

```suggestion
			return nil, microerror.Maskf(executionFailedError, "failed to read file %#q with error %#q", filePath, err)
```

To keep some context for the end user.